data/json/mapgen/bunker.json

```diff
@@ -100,6 +100,19 @@
         "M": "t_wall_metal",
         "|": "t_reinforced_glass"
       },
+      "place_monster": [
+        { "monster": "mon_mech_combat", "x": 2, "y": 2, "chance": 2 },
+        { "monster": "mon_mech_lifter", "x": 2, "y": 2, "chance": 5 },
+        { "monster": "mon_mech_recon", "x": 2, "y": 2, "chance": 3 }
+      ],
+      "place_items": [
+        { "item": "bunker_basement_loot", "x": [ 2, 8 ], "y": [ 2, 7 ], "chance": 80, "repeat": [ 1, 5 ] },
+        { "item": "bunker_basement_loot", "x": [ 2, 8 ], "y": [ 9, 14 ], "chance": 80, "repeat": [ 1, 5 ] },
+        { "item": "bunker_basement_loot", "x": [ 2, 8 ], "y": [ 16, 21 ], "chance": 80, "repeat": [ 1, 5 ] },
+        { "item": "bunker_basement_loot", "x": [ 15, 21 ], "y": [ 2, 7 ], "chance": 80, "repeat": [ 1, 5 ] },
+        { "item": "bunker_basement_loot", "x": [ 15, 21 ], "y": [ 9, 14 ], "chance": 80, "repeat": [ 1, 5 ] },
+        { "item": "bunker_basement_loot", "x": [ 15, 21 ], "y": [ 16, 21 ], "chance": 80, "repeat": [ 1, 5 ] }
+      ],
```

Comment:
Bunker basement loot was changed in https://github.com/CleverRaven/Cataclysm-DDA/pull/32486, you may want to reake it.